benchmarks/dynamo/common.py

```diff
@@ -737,11 +738,16 @@ def maybe_mark_profile(*args, **kwargs):
         for k, v in kwargs["dynamo_stats"].items():
             headers.append(k)
             row.append(v)
-    output_csv(
-        output_filename,
-        headers,
-        row,
-    )
+    if (
+        torch.distributed.is_available()
+        and torch.distributed.is_initialized()
+        and torch.distributed.get_rank() == 0
```

Comment:
use `args.init_distributed` to check first, then check torch.distributed submodules

otherwise, all benchmarks will require distributed build

Reply:
It should not break the single GPU case. If distributed is not built, `torch.distributed.is_available()` returns False and it is the very first check.